src/main/js/controls/Gantt/Track.js

```diff
@@ -16,7 +16,7 @@ import { loadActions, unloadActions } from "./helpers/actionHelpers";
 import { loadActivities, unloadActivities } from "./helpers/activityHelpers";
 import {
     createTrackContainer,
-    removeTrackContainer,
+    removeTrackContainer, scaleGraph,
```

Comment:
move the `scalegraph` next line

Reply:
[fixed](https://github.com/cerner/carbon-graphs/pull/263/commits/c0b2d8703781d7f0ae39f2330af0eed4365b8141#diff-bdf6ac5b711158c299d4a3c2dce8913fR20)